src/universal/redux/makeReducer.js

```diff
@@ -1,16 +1,12 @@
-import {compose} from 'redux';
-import {combineReducers} from 'redux-immutablejs';
+import {compose, combineReducers} from 'redux';
 import {cashayReducer} from 'cashay';
-import auth from '../modules/auth/ducks/auth';
-import meetingModule from '../modules/meeting/ducks/index';
 import notifications from '../modules/notifications/ducks/notifications';
+import {reducer as formReducer} from 'redux-form';
```

Comment:
alpha has some bugs, but overall it is a lot cleaner than the previous version. i'm a fan!